### PR TITLE
BRS-315: UI changes to Park Search

### PR DESCRIPTION
**Pull request by meredithom**
### Jira Ticket:
BRS-315

### Jira Ticket URL:
https://bcparksdigital.atlassian.net/browse/BRS-315

### Description:
Buttons, fonts, spacing and colours updated to match Figma designs. Popular filter is currently hidden on mobile due to issues with code duplication. 

BRS-315: Minor updates to spacing and colours to match Figma designs

BRS-315: Sort dropdown removed and filter column aligned with top of search results

BRS-315: Sort dropdown removed from mobile popup to match desktop version

BRS-315: Mobile Popular filter populated with filters, general file cleanup

BRS-315: Popular filters hidden on mobile, functionality partially built out but not 100% completed

BRS-315: Minor css update to reflect new Figma design for mobile.

### CODE REVIEW

**Review by DayKirby-OXD — CHANGES_REQUESTED** [review]
Looking good, nice to see this getting better.
Getting some warnings in the output would be good to clean up (unused components e.g. Select, duplicate keys e.g. setQuickSearch, etc.) so next dev doesn't see them
A few things commented out which could probably just be deleted

**Review by DayKirby-OXD — APPROVED** [review]

**Comment by meredithom** [comment]
> Looking good, nice to see this getting better. Getting some warnings in the output would be good to clean up (unused components e.g. Select, duplicate keys e.g. setQuickSearch, etc.) so next dev doesn't see them A few things commented out which could probably just be deleted

The unused Select was there before I came into the file, and the commented out code is the foundation for adding Popular Filters to mobile. I noticed I had a few errors showing up before I even got into editing. Given that we're going to be completely refactoring this page

> 

Addressed in [085defe](https://github.com/bcgov/bcparks.ca/pull/265/commits/085defed41b09c7feb11c066d1cbd68b312d28c4)

File: src/staging/src/components/search/searchFilter.js [hunk 1]
```diff
@@ -47,6 +49,20 @@ const SearchFilter = ({
     setOpenFilter(false)
   }
 
+  //Quick Search/Popular
+  const handleQuickSearchChange = event => {
```
[the remark this answers is not in the document]
Comment (meredithom) [reply src/staging/src/components/search/searchFilter.js]: [085defe](https://github.com/bcgov/bcparks.ca/pull/265/commits/085defed41b09c7feb11c066d1cbd68b312d28c4)

File: src/staging/src/components/search/searchFilter.js [hunk 2]
```diff
@@ -23,22 +23,24 @@ import ExpandMore from "@material-ui/icons/ExpandMore"
 import Select from "react-select"
 import { navigate } from "gatsby"
 
+
 const SearchFilter = ({
   data: {
     activityItems,
     facilityItems,
+    quickSearchFilters,
```
[the remark this answers is not in the document]
Comment (meredithom) [reply src/staging/src/components/search/searchFilter.js]: [085defe](https://github.com/bcgov/bcparks.ca/pull/265/commits/085defed41b09c7feb11c066d1cbd68b312d28c4)

File: src/staging/src/components/search/searchFilter.js [hunk 3]
```diff
@@ -540,17 +570,17 @@ SearchFilter.propTypes = {
   data: PropTypes.shape({
     activityItems: PropTypes.array.isRequired,
     facilityItems: PropTypes.array.isRequired,
+    quickSearchFilters: PropTypes.array.isRequired,
     openFilter: PropTypes.bool.isRequired,
     setOpenFilter: PropTypes.func.isRequired,
+    quickSearch: PropTypes.array.isRequired,
```
[the remark this answers is not in the document]
Comment (meredithom) [reply src/staging/src/components/search/searchFilter.js]: [085defe](https://github.com/bcgov/bcparks.ca/pull/265/commits/085defed41b09c7feb11c066d1cbd68b312d28c4)

File: src/staging/src/pages/explore.js [hunk 4]
```diff
@@ -615,28 +588,70 @@ export default function Explore({ location, data }) {
...
+                              InputProps={{
+                                startAdornment: (
+                                  <InputAdornment position="start">
+                                    <SearchIcon className="search-icon" />
+                                  </InputAdornment>
+                                ),
+                              }}
+                            />
+                          </div>
+                          <div class="m15t col-12 park-search-text-box-container d-none d-xl-block d-lg-block d-md-none d-sm-none d-xs-none">
```
[the remark this answers is not in the document]
Comment (meredithom) [reply src/staging/src/pages/explore.js]: [085defe](https://github.com/bcgov/bcparks.ca/pull/265/commits/085defed41b09c7feb11c066d1cbd68b312d28c4)

File: src/staging/src/pages/explore.js [hunk 5]
```diff
@@ -459,7 +463,7 @@ export default function Explore({ location, data }) {
     });
   }, [
     params,
-    sortOption,
+    //sortOption,
```
[the remark this answers is not in the document]
Comment (meredithom) [reply src/staging/src/pages/explore.js]: [085defe](https://github.com/bcgov/bcparks.ca/pull/265/commits/085defed41b09c7feb11c066d1cbd68b312d28c4)

File: src/staging/src/components/search/searchFilter.js [hunk 6]
```diff
@@ -255,34 +291,38 @@ const SearchFilter = ({
                 </div>
               </div>
               <div className="p20l-filter col-lg-8 col-md-12 col-sm-12">
-                <div className="row">
+
+                {/* TODO: Add Popular */}
+
+                {/* <div className="row">
```
[the remark this answers is not in the document]
Comment (meredithom) [reply src/staging/src/components/search/searchFilter.js]: [085defe](https://github.com/bcgov/bcparks.ca/pull/265/commits/085defed41b09c7feb11c066d1cbd68b312d28c4)

File: src/staging/src/pages/explore.js [hunk 7]
```diff
@@ -615,28 +588,70 @@ export default function Explore({ location, data }) {
...
-                        options={sortOptions}
-                        onChange={handleSortChange}
-                        placeholder="Sort by"
-                      />
+                    <div className="col-lg-8 col-md-6 col-sm-12 col-xs-12 pr10 p10t">
+                      <div className="d-block d-sm-block d-xs-block d-md-block d-lg-none d-xl-none">
+                        <Button
+                          fullWidth
+                          className="bcgov-normal-blue mobile-search-element-height h50p"
+                          onClick={() => {
```
[the remark this answers is not in the document]
Comment (meredithom) [reply src/staging/src/pages/explore.js:576]: 👍

File: src/staging/src/styles/search.scss [hunk 8]
```diff
@@ -666,11 +666,13 @@ div[class$="-singleValue"] {
 }
 
 .text-green {
-  color: green;
+  color: #008556;
```
[the remark this answers is not in the document]
Comment (meredithom) [reply src/staging/src/styles/search.scss:669]: Will be addressed in refactoring

File: src/staging/src/styles/search.scss [hunk 9]
```diff
@@ -666,11 +666,13 @@ div[class$="-singleValue"] {
...
+  font-weight: bold;
```
[the remark this answers is not in the document]
Comment (meredithom) [reply src/staging/src/styles/search.scss:670]: Will be addressed in refactoring

File: src/staging/src/pages/explore.js [hunk 10]
```diff
@@ -1011,6 +1027,7 @@ export default function Explore({ location, data }) {
                                                       <div className="park-af-list pr3">
                                                         <b>Facilities:</b>
                                                       </div>
+                                                      <div></div>
```
[the remark this answers is not in the document]
Comment (meredithom) [reply src/staging/src/pages/explore.js:1010]: Will be addressed in refactoring

**Pull request merged**